src/application/tally/widgets/organization-proposals.container.tsx

```diff
@@ -97,12 +97,13 @@ export const OrganizationProposalsContainer = ({
     onPrevious: showPreviousProposal,
     onNext: showNextProposal,
   };
-  if (!currentProposal || !tallyName) {
+  if (!currentProposal || !twitterHandle) {
```

Comment:
also this check for twitterHandle is not needed (looking at types it's always string)